neos/neos-ui
# TASK: scroll content node into view when selecting a node from content tree

At first I wanted to use a polyfill for `scrollIntoView` with animation support, but all solutions I found didn't work with iframe scrolling (duh!), so I ended up animating the `scrollTop` property of the scrolling element.

**How to test:**

Select a node from content tree and make sure the page scrolls to reveal the active content node.
